include/aspect/material_model/visco_plastic.h

```diff
@@ -255,6 +255,25 @@ namespace aspect
 
         std::vector<double> thermal_conductivities;
 
+        /**
+         * Whether conductivity should be adjusted taking into account hydrothermal circulation or not.
+         */
+        bool use_hydrothermal_conductivity;
+
+        /**
+         * temperature and depth up to which hydrothermal circulation is considered to take place.
```

Comment:
Please have a separate comment for each variable. And make sure to be consistent, e.g., start each sentence with a capital and end it with a dot. Same for other comments below.